docs/getting-started.md

```diff
@@ -0,0 +1,71 @@
+# Getting started
```

Comment:
this might be better off in the root readme

Reply:
Does linking to it work for you? I tend to find people are less likely to find or fully read instructions when they're within a really long README on the homepage. So something like

```markdown
### Getting started

Follow the instructions [here](docs/getting-started.md) to get started.
```